website/content/en/docs/concepts/disruption.md

```diff
@@ -172,7 +171,7 @@ Karpenter will add the `Drifted` status condition on NodeClaims if the NodeClaim
 Automated forceful methods will begin draining nodes as soon as the condition is met. Note that these methods blow past NodePool Disruption Budgets, and do not wait for a pre-spin replacement node to be healthy for the pods to reschedule, unlike the graceful methods mentioned above. Use Pod Disruption Budgets and `do-not-disrupt` on your nodes to rate-limit the speed at which your applications are disrupted.
 
 ### Expiration
-Karpenter will disrupt nodes as soon as they're expired after they've lived for the duration of the NodePool's `spec.template.spec.expireAfter`. You can use expiration to periodically recycle nodes due to security concern. 
+Karpenter will disrupt nodes as soon as they're expired after they've lived for the duration of the NodePool's `spec.template.spec.expireAfter`. You can use expiration to periodically recycle nodes due to security concern.
```

Comment:
Is it really the NodePool's `expireAfter` or the NodeClaim's `expireAfter`? Consider that the NodePool could be updated after a Node/NodeClaim was created and the `expireAfter` is changed.

Reply:
You're right, it would be the field on the NodeClaim rather than the NodePool. We have a few artifacts from when `expireAfter` wasn't persisted to the NodeClaim. I'll take a look and see just how many there are, if it's only a few I'll scope them in here otherwise I'll probably open a separate PR.